style.css

```diff
@@ -1,54 +1,77 @@
 * {
   margin: 0;
   padding: 0;
+
+  /* outline: 2px solid #f80606;  */
```

Comment:
[OPTIONAL]
- [ ] Kindly consider removing the commented block of code in your code base as it's not the best practice for clean code.